### PR TITLE
Update README with links to Notebooks

### LINKED ISSUE #60: overview of folder structure
We need an overview of where everything is, and where to start.

@DANAJK you suggested to have this in a notebook. I think I'd prefer to have it part of the README (potentially splitting that up in several notebooks). You can open those in the `jupyter` interface with reasonable formatting (although sadly links etc don't work). What do you think?

We could have both of course. Maybe update frequency is low enough that duplication doesn't matter.

If a notebook, I guess it would sit in the `Introductory` folder.

### PR DESCRIPTION
**Pull request by ashgillman**
Is this what you had in mind to fix #60?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ Getting started
 ================
 Some exercises use data that you will need. Check the [installation instructions](INSTALL.md).
 
+If you are using the Docker `service` image (if you're in a course using a cloud service, this is the case), the following steps are automatically run for you. Skip ahead to [Docker](#docker)/[Cloud](#cloud).
+
```
Comment (KrisThielemans) [inline README.md:54]: this is incorrect. I'm revising that stuff so please undo this change

File: README.md [hunk 2]
```diff
@@ -91,8 +93,9 @@ which should start your web-browser automatically.
 
 ### Docker
 
-The jupyter notebook should already be started for you. Follow instructions in our
+The jupyter notebook should already be started for you if you are using the `service` image. Follow instructions in our
```
Comment (KrisThielemans) [inline README.md:96]: undo

File: README.md [hunk 3]
```diff
@@ -91,8 +93,9 @@ which should start your web-browser automatically.
...
 [Docker README](https://github.com/SyneRBI/SIRF-SuperBuild/tree/master/docker#readme).
+If prompted for a password, it is `virtual`.
```
Comment (KrisThielemans) [inline README.md:98]: undo

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
let's keep the last changes only.

**Comment by KrisThielemans** [comment]
competing PR #130. Let's merge this one first after the above adjustments.

File: README.md (hunk 1 above)
Comment (ashgillman) [reply README.md:54]:
```suggestion

```

File: README.md (hunk 2 above)
Comment (ashgillman) [reply README.md:96]:
```suggestion
The jupyter notebook should already be started for you. Follow instructions in our
```

File: README.md (hunk 3 above)
Comment (ashgillman) [reply README.md:98]:
```suggestion
```
